content/en/docs/setup/install/multicluster/primary-remote_multi-network/snips.sh

```diff
@@ -124,7 +119,7 @@ NAME                    TYPE           CLUSTER-IP    EXTERNAL-IP    PORT(S)   AG
 istio-eastwestgateway   LoadBalancer   10.0.12.121   34.122.91.98   ...       51s
 ENDSNIP
 
-snip_expose_services_in_cluster2_1() {
-kubectl --context="${CTX_CLUSTER2}" apply -n istio-system -f \
+snip_expose_services_in_cluster1_and_cluster2_1() {
```

Comment:
Key changes for the test are `snip_expose_services_in_cluster1_1` and `snip_expose_services_in_cluster2_1` are removed in the snips replaced by `snip_expose_services_in_cluster1_and_cluster2_1`.